elements/bulbs-video/elements/carousel-slider.js

```diff
@@ -0,0 +1,128 @@
+import { BulbsHTMLElement, registerElement } from 'bulbs-elements/register';
+
+import './carousel-slider.scss';
+
+export default class CarouselSlider extends BulbsHTMLElement {
+  createdCallback () {
+    this.currentIndex = 0;
+    this.track = document.createElement('bulbs-video-carousel-track');
+    while (this.firstChild) {
+      this.track.appendChild(this.firstChild);
+    }
+    this.appendChild(this.track);
+  }
+
+  attachedCallback () {
+    this.slideItems();
+  }
+
+  get carouselItems () {
+    return this.track.children;
+  }
+
+  getGridRatio () {
+    if (this.carouselItems[0]) {
+      return (this.getItemWidth() / this.offsetWidth) || 0;
+    }
+    else {
+      return 0;
+    }
+  }
+
+  getItemMargin () {
+    let child = this.carouselItems[0];
```

Comment:
A couple property definitions above at `get carouselItems () {`

Now could be a good moment to discuss use of `get` and `set`.

Here I've made a lot of getters be methods because they actually do some DOM access and computation, but the `carouselItems` getter is just an alias for property access through the track property.

Does it make sense that way?